.ci/install_kata_kernel.sh

```diff
@@ -72,16 +85,33 @@ build_and_install_kernel() {
 	popd
 }
 
+install_packaged_kernel(){
+	if [ "$ID"  == "ubuntu" ]; then
+		sudo apt install -y "$packaged_kernel"
+	elif [ "$ID"  == "fedora" ]; then
+		sudo dnf install -y "$packaged_kernel"
+	elif [ "$ID"  == "centos" ]; then
+		sudo yum install -y "$packaged_kernel"
+	else
+		die "Unrecognized distro"
+	fi
+}
+
 cleanup() {
 	rm -rf "${tmp_dir}"
 }
 
 main() {
-	download_repo
-	kernel_version=$(get_kernel_version)
-	download_kernel ${kernel_version}
-	build_and_install_kernel ${kernel_version}
-	cleanup
+	current_kernel_version=$(get_current_kernel_version)
+	packaged_kernel_version=$(get_packaged_kernel_version)
+	if [ "$packaged_kernel_version" == "$current_kernel_version" ] && [ "$kernel_arch" == "x86_64" ]; then
```

Comment:
correct me if I am wrong but I think this wont guarantee that the kernel has the same configuration and patches. 

For this we will need a way to know the status/version of the kernel configuration (config and patches). 

One way (what we did in the past is to have a file that we need to increment every time we update the kernel and config. We could add that number as part of the package version. And append it to the extra kernel version (to verify when uname -a is used).

Reply:
```
packaging/
--> kernel/
     ----> kernel.yaml
```
```
patchset_version=1
```
If a patch/config is added or modified we bump patch-set version.

If check if the packaged kernel version is (kernel_version-patchset-version-kernel_pkg-version) if not build from source

Reply:
@chavafg  https://github.com/kata-containers/packaging/pull/46 now adds an extra number to the kernel package version.